istioctl/cmd/auth.go

```diff
@@ -95,7 +100,40 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 		},
 	}
 
-	// TODO(phillip): Add the upgrade command for the new authorization v1beta1 policy.
+	upgradeCmd = &cobra.Command{
+		Hidden: true,
+		Use:    "upgrade",
+		Short:  "Upgrade Istio Authorization Policy from version v1alpha1 to v1beta1",
+		Long: `Upgrade converts Istio authorization policy from version v1alpha1 to v1beta1. It requires access to Kubernetes
+service definition in order to translate the service name specified in the ServiceRole to the corresponding
+workload labels in the AuthorizationPolicy. The service definition could be provided either from the current
+Kubernetes cluster or from a yaml file specified from command line.
+
+THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
+`,
+		Example: `  # Upgrade the Istio authorization policy with service definition from the current k8s cluster:
```

Comment:
Use `Kubernetes`, not `k8s`.  See https://istio.io/about/contribute/style-guide/#miscellaneous .  Change this everywhere k8s is in a string that will be surfaced to a user or log.